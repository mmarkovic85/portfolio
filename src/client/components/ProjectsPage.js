import React from "react";

const ProjectsPage = () => (
  <h1>Projects page placeholder</h1>
);

export {
  ProjectsPage as default
};
