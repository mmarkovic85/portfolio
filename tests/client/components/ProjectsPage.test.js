import React from "react";
import renderer from "react-test-renderer";
import ProjectsPage from "../../../src/client/components/ProjectsPage";

test("Should render ProjectsPage component correctly", () => {
  const rendered = renderer.create(
    <ProjectsPage />
  );

  expect(rendered.toJSON()).toMatchSnapshot();
});
