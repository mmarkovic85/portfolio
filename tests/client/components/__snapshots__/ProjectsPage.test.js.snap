// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ProjectsPage component correctly 1`] = `
<h1>
  Projects page placeholder
</h1>
`;
